doc/md/Installation_Instructions/Windows-Installation-Instructions.md

```diff
@@ -135,4 +135,12 @@ In later versions of windows 10 you may be able to get color to work by setting
 [HKEY_CURRENT_USER\Console]
     "VirtualTerminalLevel"=dword:00000001
 ```
-If after setting this key (and restarting proxmark3.exe) you get extra characters and no color text, either delete the key or set the value to 0
+You also need to disable "use legacy console" in the cmd.exe properties, or set the following registry key
+```
+[HKEY_CURRENT_USER\Console]
+    "ForceV2"=dword:00000001
+```
+After making these changes, you will need to start a new command prompt (cmd.exe) to ensure its using the new settings.
+
+If after making these changes (and restarting proxmark3.exe) you get extra characters and no color text, set either key to 0 or enable legacy mode again (and restart the command prompt).
```

Comment:
last sentence: compared to the old sentence (if it doesn't work, delete VTL key or set key=0), this sentence seems to talk only about second key FV2.
I guess this should be rephrased such as "restore to the initial situation by either deleting the keys or setting their value to 0"

Reply:
I will have a think and reword.

With the new code if 'either' of them are not set then the it will not enable color. (this will catch the users that have the legacy unchecked for some other reason.
i.e. I am treating them as prerequisites for color to be on.
So "set either key to 0" will turn off color, OR enable legacy mode (which is the user friendly way to turn it off.